src/bin/rav1e.rs

```diff
@@ -137,13 +137,34 @@ fn main() {
     video_info.time_base.num as usize
   );
 
+  if cli.limit != 0 {
+    cli.limit = cli.limit - cli.skip ;
+  }
+
   let mut progress = ProgressInfo::new(
     Rational { num: video_info.time_base.den, den: video_info.time_base.num },
-    if cli.limit == 0 { None } else { Some(cli.limit) },
+    if cli.limit == 0 && cli.skip == 0 {
```

Comment:
please follow the coding style

Reply:
the cli.skip check doesn't do anything

Reply:
Will remove this parameter,
Thought of keeping it to double check the values before showing but we don't need it, now only I realised.